airflow/example_dags/example_passing_params_via_test_command.py

```diff
@@ -30,18 +30,15 @@
 
 
 @task(task_id="run_this")
-def my_py_command(params, test_mode=None, task=None):
```

Comment:
It wasn't clear to me if a specific case was intended to be tested here originally with the usage of the `params` name here, which is also a name of the `task.params` attribute or it's just a coincidence.

Moved it to a simpler structure. If the original one made sense and is needed as well, please let me know, I'll restore it.